pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>api-sdk-manager-java-library</artifactId>
             <version>1.0.4</version>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.companieshouse</groupId>
+            <artifactId>private-api-sdk-java</artifactId>
+            <version>2.0.89</version>
```

Comment:
Latest version is 2.0.147 - https://github.com/companieshouse/private-api-sdk-java/releases/tag/2.0.147. Any reason not to go for the latest?